CMakeLists.txt

```diff
@@ -956,7 +974,6 @@ IF(PCRE2_BUILD_TESTS)
   ENDIF(MSVC)
 
   ADD_EXECUTABLE(pcre2test ${PCRE2TEST_SOURCES})
-  SET(targets ${targets} pcre2test)
```

Comment:
this is a valid production target, unike the others

Reply:
Fixed.